aws/cloudformation/lambda.yml.erb

```
@@ -273,6 +273,48 @@ Resources:
             - "ec2:DescribeImages"
             Resource: '*'
       PermissionsBoundary: !ImportValue IAM-DevPermissions
+
+  SQLUserLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Description: 'Custom CloudFormation Resource: Create/Update/Delete a SQL User and its permissions in a MySQL database.'
+      FunctionName: !Sub "SQLUser-CloudFormation-Custom-Resource-${AWS::StackName}"
+      Code: <%=package_node_lambda 'sql-user' %>
+      Handler: index.handler
+      Runtime: nodejs18.x
+      MemorySize: 1024
+      Timeout: 30
+      Role: !GetAtt SQLUserRole.Arn
+      VpcConfig:
+        SecurityGroupIds: [!ImportValue VPC-FrontendSecurityGroup]
+        SubnetIds: <%=subnets.to_json%>
+  SQLUserRole:
```

Minor nit on naming, out of context if I was looking at a list of IAM Roles it wouldn't be immediately clear what this is for.

```suggestion
  SQLUserLambdaRole:
```